### PR TITLE
Add three level toc in left side bar

### PR DESCRIPTION
**Pull request by netroby**
see #31

### CODE REVIEW

**Comment by netroby** [comment]
ping @d0ugal  @waylan @tomchristie @jimporter @juzraai

File: src/_base/css/font-awesome-4.0.3.css [hunk 1]
```diff
@@ -1336,3 +1336,10 @@
 .fa-plus-square-o:before {
   content: "\f196";
 }
+.lilv2 {
+    padding-left:24px;
+}
+.lilv3 {
+    padding-left:48px;
+}
```
Comment (jimporter) [inline src/_base/css/font-awesome-4.0.3.css]: These changes should go in `base.css` (and you'll want to put a space after the colon).

**Comment by netroby** [comment]
I am not professional python developer. this is just my modified theme for myself document project.
Hope you will add it into mkdocs theme.

Thanks

**Comment by waylan** [comment]
The changes mentioned by @jimporter need to be made before this can be accepted.

> You'll also want to think about how to make this configurable by users

You may want to take a look at the upcoming MkDocs release which adds support for [theme specific settings][1]. Actually, these themes will need to be updated to work with the next release.

[1]: https://github.com/mkdocs/mkdocs/blob/master/docs/user-guide/custom-themes.md#theme-configuration

**Pull request closed without merging**